src/sentry/data_secrecy/service/model.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+from django.utils import timezone
+from pydantic import Field
+
+from sentry.hybridcloud.rpc import RpcModel
+
+
+class RpcDataSecrecyWaiver(RpcModel):
+    organization_id: int = -1
+    access_start: datetime = Field(default_factory=timezone.now)
+    access_end: datetime = Field(default_factory=timezone.now)
```

Comment:
Oof, not ideal that we're going to call `timezone.now()` twice during object initialization and that we might get unequal timestamps (potentially with the end before the start). It's very unlikely to matter; just bugs me.

If you're interested in a bit of hacking, you could look into fixing it by overriding `__init__` (and manually calling `timezone.now()` to provide a common value to both attrs). I don't recommend this unless it sounds intrinsically rewarding; just thought it was interesting.